src/highdicom/io.py

```diff
@@ -225,12 +225,14 @@ class ImageFileReader(object):
     Examples
     --------
     >>> from highdicom.io import ImageFileReader
-    >>> with ImageFileReader('/path/to/file.dcm') as image:
+    >>> with ImageFileReader('data/test_files/ct_image.dcm') as image:
     ...     print(image.metadata)
     ...     for i in range(image.number_of_frames):
     ...         frame = image.read_frame(i)
     ...         print(frame.shape)
-
+    Dataset.file_meta -------------------------------
+    (0002, 0000) File Meta Information Group Length  UL: 192
+    ...
```

Comment:
The motivation behind the ImageFileReader is to read individual frames from multiframe DICOM files, so using a single frame DICOM file like this is not a good example. I suggest we switch to an example multiframe CT from the pydicom test data, which has two frames to iterate over:

```python
>>> from pydicom.data import get_testdata_file
>>> test_filepath = get_testdata_file('eCT_Supplemental.dcm')
>>>
>>> with ImageFileReader(test_filepath) as image:
...     print(image.metadata.SOPInstanceUID)
...     for i in range(image.number_of_frames):
...         frame = image.read_frame(i)
...         print(frame.shape)
1.3.6.1.4.1.5962.1.1.10.3.1.1166562673.14401
(512, 512)
(512, 512)
```